hack/chains/kaniko-cosign-verify.sh

```diff
@@ -35,7 +35,7 @@ title "Cosign verify the image"
 # Save the output data to a file so we can look at it later
 # (Actually we could just pipe it to jq because the text goes to stderr I think..?)
 show-then-run "cosign verify --key $SIG_KEY $IMAGE_URL --output-file /tmp/verify.out"
-yq e -P /tmp/verify.out
+cat /tmp/verify.out | yq e . -P -
```

Comment:
Consider using this form instead: `< /tmp/verify.out yq e . -P -`

It works just fine on my fedora which has `yq` installed from snap.

Reply:
Hmm that works, but looks weird to me.

I think cat is okay, but there's some value in avoiding cat I guess, so what about this:

    yq e . -P - < /tmp/verify.out

Same thing but slightly less weird looking IMO.